parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -355,8 +374,16 @@ async def folder_create(self, path: FsPath) -> EntryID:
             await self.local_storage.set_manifest(parent.id, new_parent)
 
         # Send events
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=child.id)
+        self._send_event(
+            CoreEvent.FS_ENTRY_UPDATED, id=parent.id, reason=FSEntryUpdatedReason.FOLDER_CREATE
+        )
```

Comment:
I would add the `entry_id`/`entry_name` fields in this event instead of `FOLDER_CREATE_ENTRY_CREATION`.
The rational is this event is the "main" one (the one we are actually listening for if we are interested in changes in a given directory), the `FOLDER_CREATE_ENTRY_CREATION` is only interesting if we are listening for any changes (like what does the sync monitor)